main/src/ca/uwaterloo/flix/language/ast/AtomicOp.scala

```diff
@@ -48,6 +48,12 @@ object AtomicOp {
 
   case class RecordRestrict(label: Name.Label) extends AtomicOp
 
+  case class VariantIs(label: Name.Label) extends AtomicOp
```

Comment:
I think `ExtensibleIs` is better since both tag operate on variants

Reply:
Good point; fixed.